README.md

```diff
@@ -1,2 +1,10 @@
 # DataScience-2019-PUSK
-This repository was created for final project at Data Science course by Alexander Duymin in MEPhI 2019 (7th term). Contributors are: Panfilov Ivan, Usman Dmitriy, Smirnov Alexander, Kutuzov Artem (PUSK)
+This repository was created for final project at Data Science course by Alexander Duymin in MEPhI 2019 (7th term). Contributors are: Panfilov Ivan, Усман Дмитрий, Smirnov Alexander, Kutuzov Artem (PUSK)
```

Comment:
Why did you change your first and last name from English to Russian? If I made a mistake writing them just correct as it is necessary